runtime/recipe/recipe.js

```diff
@@ -24,6 +25,9 @@ class Recipe {
     // can handle. ConnectionConstraints should be a different record
     // type to particles/views.
     this._connectionConstraints = [];
+
+    // ??? should this be an array maybe?
```

Comment:
Not sure yet. Make this a TODO? I think there are possibilities when merging recipes together that we might encounter multiple search strings.

Reply:
done.
and I actually noticed that merging 2 recipes both with search string won't work well, so I tweaked the implementation (and added a test)